bindings/pydeck/requirements/requirements.txt

```diff
@@ -1,5 +1,3 @@
 ipykernel>=5.1.2;python_version>="3.4"
-ipywidgets>=7.0.0
-traitlets>=4.3.2
```

Comment:
Removing this from `requirements.txt` does nothing for a local (and I assume CI) build of pydeck because `make init` also installs `requirements-dev.txt` and we depend on the `jupyter` metapackage there:
https://github.com/visgl/deck.gl/blob/e275028f7907a14bfca021d5c65de3bd93b79f5c/bindings/pydeck/requirements/requirements-dev.txt#L1